pkg/virt-operator/install-strategy/create.go

```diff
@@ -3358,3 +3264,56 @@ func createOrUpdateKubeVirtCAConfigMap(
 	}
 	return nil, nil
 }
+
+func removeKvServiceAccountsFromDefaultSCC(targetNamespace string,
+	clientset kubecli.KubevirtClient,
+	stores util.Stores) error {
+
+	modified := false
```

Comment:
Just a nit: I'd prefer to set this variable closer to the place where you're going to use it. Just to make it easier to find.

Reply:
@danielBelenky Thank you. Done.